README.md

```diff
@@ -38,6 +45,14 @@ Brion provides the following major features:
   [Lunchbox](http://eyescale.github.io/Lunchbox-1.8/index.html),
   [Boost](http://www.boost.org/doc/libs).
 
+## Algorithmic library
```

Comment:
Is it really algorithmic? It takes man years to understand the data model, but in the end it's more a convenience/high-level library.